Mathlib/Order/KrullDimension.lean

```diff
@@ -27,6 +32,17 @@ sum of `-∞` and the Krull dimension of any other varieties.
 -/
 
 variable (α : Type*) [Preorder α]
+variable {β : Type _} (r : Rel β β)
+
+/--
+Krull dimension of a set `α` with a binary relation `r` is the supremum of the rightmost index of
+all relation series of `r`. If there is no relation series `a₀, a₁, ..., aₙ` in `α`, then its Krull
+dimension is defined to be negative infinity; if the length of all relation series `a₀, a₁, ..., aₙ`
+is unbounded, its Krull dimension is defined to be positive infinity. (Not sure if this definition
+is useful.)
```

Comment:
Based on the above comment, can I ask why you include this definition?

Reply:
`krullDimOfRel` seems not to be a very useful definition. The only other relation I can think of that we want to consider Krull dimension of is `covby` for submodules. The length of a module can be defined as krull dimension of the poset submodules; the length of a composition series; the krull dimension of `covby` relation. But the last is conceptually the same as length of the composition series. So probably `krullDimOfRel` is not too useful a definition so I think we should remove it.

Reply:
Thanks; then I agree that this could be deleted.